internal/engine/buildcontrol/build_control.go

```diff
@@ -92,6 +92,9 @@ func RemoveTargetsWithBuildingComponents(mts []*store.ManifestTarget) []*store.M
 	for _, mt := range mts {
 		if mt.State.IsBuilding() {
 			building[mt.Manifest.ID()] = true
+
+			// TODO(nick): This logic isn't quite right.  A manifest can re-use image
```

Comment:
But we still don't want them to build in parallel, or we might try to build the same image twice at the same time, right? Or am I missing something?

Reply:
ah yeah that makes sense!